pkg/cloud/services/ec2/securitygroups.go

```diff
@@ -430,7 +430,10 @@ func (s *Service) getDefaultSecurityGroup(role infrav1.SecurityGroupRole) *ec2.S
 }
 
 func (s *Service) getSecurityGroupTagParams(name string, id string, role infrav1.SecurityGroupRole) infrav1.BuildParams {
-	additional := infrav1.Tags{}
+	additional := s.scope.AWSCluster.Spec.AdditionalTags
+	if additional == nil {
```

Comment:
Any particular reason we need a nil check here and you aren't doing it for the previous uses (that weren't already setting additional tags)?